pyspecdata/core.py

```diff
@@ -7663,6 +7663,21 @@ def bootstrap(self,points,swap_out = np.exp(-1.0),seedval = 10347,minbounds = {}
         print(r'\end{verbatim}')
         return recordlist # collect into a single recordlist np.array
     def set_guess(self,dict_of_values):
+        """sets parameters to guess/estimated value to compare fit.
+        
+        Parameters
+        ----------
+        dict_of_values: **kwargs 
```

Comment:
you should say `dict` (which is the type) not `**kwargs` (the `**` is a special syntax for function arguments)>

Reply:
You should also specify (either here or below) that the keys  of this dictionary need to be sympy symbols